esmvaltool/namelist.py

```diff
@@ -38,6 +39,36 @@ def read_namelist_file(filename, config_user, initialize_tasks=True):
         raw_namelist, config_user, initialize_tasks, namelist_file=filename)
 
 
+def check_ncl_version():
+    """Check the NCL version"""
+    ncl = which('ncl')
+    if not ncl:
+        raise NamelistError("Namelist contains NCL scripts, but cannot find "
+                            "an NCL installation.")
+    try:
+        cmd = [ncl, '-V']
+        version = subprocess.check_output(cmd, universal_newlines=True)
+    except subprocess.CalledProcessError:
+        logger.error("Failed to execute '%s'", ' '.join(' '.join(cmd)))
+        raise NamelistError("Namelist contains NCL scripts, but your NCL "
+                            "installation appears to be broken.")
+
+    version = version.strip()
+    logger.info("Found NCL version %s", version)
+
+    if version == "6.3.0":
+        logger.error("NCL version " + version + " not supported due to a bug "
+                     + "(see Known Issues in the ESMValTool user guide)")
+
+    if int(version.split(".")[0]) < 6:
+        logger.error("NCL version " + version +
+                     " not supported, need version 6.2.0 or higher")
+
+    if int(version.split(".")[0]) == 6 and int(version.split(".")[1]) < 2:
```

Comment:
In the new ESMValTool version we will support only NCL 6.4 or later (see also #191).